src/components/TypeLocationsPaper.js

```diff
@@ -184,8 +190,17 @@ const StyledResultPane = withTheme(withStyles(styles)(ResultPane));
 class TypeLocationsPaper extends Component {
   render() {
     const { classes, rights, title, onRefresh, onEdit, readOnly, location, ...others } = this.props;
+    const createRegionLocationRight = Boolean(
+      this.props?.rights.includes(RIGHT_REGION_LOCATION_ADD)
```

Comment:
There is no need to use _Boolean()_, because _.includes()_ is returning boolean value by default.